src/Components/CreateImageWizardV2/steps/FileSystem/FileSystemConfiguration.tsx

```diff
@@ -163,6 +164,12 @@ const getSuffix = (mountpoint: string) => {
 
 const Row = ({ partition }: RowPropTypes) => {
   const [units, setUnits] = useState<Units>('MiB');
+  const dispatch = useAppDispatch();
+
+  // @ts-ignore
+  const handleRemovePartition = (id: uuidv4) => {
```

Comment:
Let's solve the problem so we don't have to silence it here. When you remove the `@ts-ignore` the message will say that a value is used as a type. `id` is initiated as a `uuidv4`, but the type of uuid is a string. So this should fix the problem :)
```suggestion
  const handleRemovePartition = (id: string) => {
```

Reply:
@regexowl what about preventing the usage of ts-ignore?

Reply:
I mean with some rules for the linter.

Reply:
Thanks for the inspiration! :smile: https://github.com/osbuild/image-builder-frontend/pull/1799